src/test/ui/self/suggest-self-2.stderr

```diff
@@ -0,0 +1,28 @@
+error[E0425]: cannot find function `bar` in this scope
+  --> $DIR/suggest-self-2.rs:5:9
+   |
+LL |         bar(self);
+   |         ^^^ help: try calling method instead of passing `self` as parameter: `self.bar`
+
+error[E0425]: cannot find function `bar` in this scope
+  --> $DIR/suggest-self-2.rs:10:9
+   |
+LL |         bar(&self);
+   |         ^^^ help: try calling method instead of passing `self` as parameter: `self.bar`
```

Comment:
It would be nice if the suggestion rewrote the entire call:
```
try calling `bar` as a method: `self.bar();`
```

Reply:
Yeah, I was thinking about it but I'm not sure how I can do that. I was thinking that we can do something like: `Span(args[1].span.lo, args.last().span.hi)` but then I don't know how to get the actual code for that span.

Reply:
Yes, I think you'll want to do something similar to that. I haven't got time to work it out exactly now, but I'll try to get back to you if you can't figure it out.

Reply:
Sorry for the delay. I've managed to include arguments into suggestion.